composer/algorithms/augmix/README.md

```diff
@@ -0,0 +1,131 @@
+# 🎨 AugMix
+
+[\[How to Use\]](#how-to-use) - [\[Suggested Hyperparameters\]](#suggested-hyperparameters) - [\[Technical Details\]](#technical-details) - [\[Attribution\]](#attribution)
+
+`Computer Vision`
+
+For each data sample, AugMix creates an _augmentation chain_ by sampling `depth` image augmentations from a set (e.g. translation, shear, contrast).
+It then applies these augmentations sequentially with randomly sampled intensity.
+This is repeated `width` times in parallel to create `width` different augmented images.
+The augmented images are then combined via a random convex combination to yield a single augmented image, which is in turn combined via a random convex combination sampled from a Beta(`alpha`, `alpha`) distribution with the original image.
+
+| ![AugMix](https://storage.googleapis.com/docs.mosaicml.com/images/methods/aug_mix.png) |
+|:--:
+|*An image of a turtle that undergoes three different augmentation chains, gets combined back together using a convex combination, and gets combined with the original image. [Figure 4 from Hendrycks et al. (2020)](https://arxiv.org/abs/1912.02781).*|
+
+## How to Use
+
+### Functional Interface
+
+```python
+import composer.functional as cf
+from composer.algorithms.utils import augmentation_sets
+
+augmixed_image = cf.augmix_image(img=image,
+                                 severity=3,
+                                 width=3,
+                                 depth=-1,
+                                 alpha=1.0,
+                                 augmentation_set=augmentation_sets["all"])
+```
+
+### Torchvision Transform
+
+```python
+import torchvision.transforms as transforms
+import torchvision.datasets.VisionDataset as VisionDataset
+
+from composer.algorithms.augmix import AugmentAndMixTransform 
+
+augmix_transform = AugmentAndMixTransform(
+    severity=3,
+    width=3,
+    depth=-1,
+    alpha=1.0,
+    augmentation_set="all"
+)
+composed = transforms.Compose([augmix_transform, transforms.RandomHorizontalFlip()])
+dataset = VisionDataset(data_path, transform=composed)
+```
+
+### Composer Trainer
+
+TODO(MATTHEW): Fix and provide commentary and/or comments
+
+```python
```

Comment:
@ajaysaini725 - Please add a bit of text at the beginning explaining what's happening.